torch/quantization/fx/_equalize.py

```diff
@@ -244,11 +249,54 @@ def get_op_node_and_weight_eq_obs(
         return op_node, weight_eq_obs
 
     elif op_node.op == 'call_function':
-        # TODO
-        return None, None
+        weight_node = maybe_get_functional_weight_node(op_node, modules, 'eq_obs')
+        if weight_node is not None:
+            weight_eq_obs = modules[str(weight_node.target)]
+            assert(isinstance(weight_eq_obs, _WeightEqualizationObserver))
+            return op_node, weight_eq_obs
 
     return None, None
 
+def maybe_get_functional_weight_node(op_node: Node, modules: Dict[str, nn.Module], attr: str) -> Optional[Node]:
```

Comment:
I feel this function is a bit too complicated, is it possible to write a function that traces back one step, and for different cases we just call this function once, twice and three times?

Reply:
I updated the code -- does this look better?